gameserver/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:22.04
+
+ARG GODOT_VERSION="4.5.1-stable"
+ARG USERNAME=godot
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install dependencies and create user
+RUN apt-get update && apt-get install -y \
+    wget \
+    unzip \
+    dotnet-sdk-8.0 \
+    ca-certificates \
+    libfontconfig1 \
+    && groupadd --gid $USER_GID $USERNAME \
+    && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Godot (Mono version)
+RUN wget -q https://github.com/godotengine/godot/releases/download/${GODOT_VERSION}/Godot_v${GODOT_VERSION}_mono_linux_x86_64.zip \
```

Comment:
We use wn godot version (compiled in double precision for float 64 bits): https://github.com/DyingStar-game/godotandaddons/releases/tag/4.5.1-stable